words_dir/tweet_labels.py

```diff
@@ -10,14 +10,14 @@
 labels = {
     # 'label': 'defintion'
     # sexual orientation
-    'gay': '',
-    'lesbian': '',
-    'bisexual': '',
-    'pansexual': '',
+    'gay': 'Possessing sexual and/or emotional attraction towards members of the same sex.',
+    'lesbian': '(of a woman) Homosexual, gay; preferring mostly or exclusively women as romantic or sexual partners.',
```

Comment:
if label for gay is tweaked, I suggest taking out '(of a woman) Homosexual, gay'

Reply:
If I remove "(of a woman)" I think it should still say somewhere in the definition that it is about women that prefer mostly or exclusively women partners, do you have a suggestion how to go forward about that?

Reply:
I agree with you! I found a better definition [here](https://gaycenter.org/about/lgbtq/), it define the label lesbian as 'A woman whose enduring physical, romantic, and/or emotional attraction is to other women. Some lesbians may prefer to identify as gay or as gay women.' I think this is the best that covers it.

Reply:
I've added that definition !